Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt

```diff
@@ -205,31 +210,40 @@ private fun MultipleChoiceQuestion(
                 val selectedOption = answer?.answersStringRes?.contains(option.value)
                 mutableStateOf(selectedOption ?: false)
             }
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(vertical = 4.dp)
-                    .clickable(
-                        onClick = {
-                            checkedState = !checkedState
-                            onAnswerSelected(option.value, checkedState)
-                        }
-                    )
+            Surface(
+                shape = MaterialTheme.shapes.small,
+                border = BorderStroke(
+                    width = 1.dp,
+                    color = MaterialTheme.colors.onSurface.copy(alpha = 0.12f)
+                ),
+                modifier = Modifier.padding(vertical = 4.dp)
             ) {
-                Checkbox(
-                    checked = checkedState,
-                    onCheckedChange = { selected ->
-                        checkedState = selected
-                        onAnswerSelected(option.value, selected)
-                    },
-                    colors = CheckboxConstants.defaultColors(
-                        checkedColor = MaterialTheme.colors.primary
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .clickable(
```

Comment:
Ditto `InteractionState` comment.

Reply:
this isn't possible right now: [see issue](https://issuetracker.google.com/171819073)

Reply:
ack